.github/workflows/build+release.yml

```diff
@@ -28,26 +28,39 @@ jobs:
               uses: bisgardo/github-action-regex-parse@v1
               id: tag
               with:
-                  pattern: "(?<name>(?<package>.*?)/(?<version>.*))"
+                  pattern: "(?<name>(?<package>.*?)/(?<version>.*))" # exports 'name', 'package', and 'version'
                   input: "${{steps.ref.outputs.ref-name}}"
             - name: Checkout project
               uses: actions/checkout@v3
               with:
                 ref: "${{steps.ref.outputs.ref}}"
             - name: Extract tag message
+              uses: bisgardo/github-action-echo@v1
               id: tag-msg
-              run: |
-                  msg="$(git for-each-ref "${{steps.ref.outputs.ref}}" --format='%(contents)')"
-                  echo "msg<<EOT
-                  ${msg}
-                  EOT" >> "${GITHUB_OUTPUT}"
+              with:
+                  msg: '$(git for-each-ref "${{steps.ref.outputs.ref}}" --format="%(contents)")'
             - name: Fail if tag is not "annotated" or its message is empty
               if: "steps.tag-msg.outputs.msg == ''"
               run: exit 1
-            - name: Check that version matches 'package.json' up to build version
+            - name: Collect package data
+              uses: bisgardo/github-action-echo@v1
+              id: package
+              with:
+                  name: "${{steps.tag.outputs.package}}"
+                  workspace: "@concordium/${name}"
+                  version: '$(npm pkg get version --workspaces | jq -r ".[\"${workspace}\"]")'
+                  path: "./packages/${name}"
+            - name: Collect release data
+              uses: bisgardo/github-action-echo@v1
+              id: release
+              with:
+                  _name: "${{steps.package.outputs.name}}"
+                  version: "${{steps.tag.outputs.version}}"
+                  npm-pack-file: "concordium-${_name}-${{steps.package.outputs.version}}.tgz"
+                  github-release-name: "${_name}: ${version}"
```

Comment:
I don't really understand why the action is called `...echo` :thinking:
(I would expect it to display values not define variables :upside_down_face: )

Reply:
Good point, maybe it makes more sense when you know that you need to do `echo key=value >> $GITHUB_OUTPUT` to define an output? Which is pretty crazy (and is the *simple* case of what the action hides).

It's also intended to convey that the expression is evaluated in a shell, literally as an argument to `echo` and thus is able to execute arbitrary commands.

Ideas for better names are very welcome :)